### PR TITLE
Add lint on push and PR

### PR DESCRIPTION
**Pull request by samoylenko**
Convert to NodeJS project
Add lint-md
Fix all lint issues
Add Github workflow with Lint step

Checkout the Actions tab now: https://github.com/devsecops-learning/docs/actions
More at our meeting

### CODE REVIEW

**Review by radaken — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# docs
\ No newline at end of file
+# Public team documents
+
+A first public repository
```
Comment (Fastex26rus) [inline README.md]: why are the red dots?

File: docs/getting-started.md [hunk 2]
```diff
@@ -22,10 +22,11 @@ required) list that will help join our projects:
 In our work, we use multiple online platforms, and you may want to ensure you
 registered / have an account at:
 
-- [Microsoft live](https://live.com) (to share documents, access Teams and
-  Azure)
-- [GitHub](https://github.com) (to collaborate on our projects and code)
-- [Docker Hub](https://hub.docker.com/) (to access and publish container images)
```
Comment (Fastex26rus) [inline docs/getting-started.md:30]: why is the code highlighted in red?

**Review by Fastex26rus — APPROVED** [review]

**Review by radaken — APPROVED** [review]

File: docs/getting-started.md (hunk 2 above)
Comment (radaken) [reply docs/getting-started.md:30]: cause it is deleted text

**Review by sashablacky — APPROVED** [review]
that is a lot of lines of code! great job!

File: README.md (hunk 1 above)
Comment (samoylenko) [reply README.md]: DId you mean red `docs`? Red are the lines that were removed. Green is lines that were added. Even if one character is replaced in the line, the entire line becomes red, and is getting replaced with a new line which is the same but one char. That's how git works. But also this is very useful to visually see the difference between what was there, and the new change

File: docs/getting-started.md (hunk 2 above)
Comment (samoylenko) [reply docs/getting-started.md:30]: Yep that's deleted text. Just like the previous one. Takes some time to get used to the difference view, this is a good  find

**Pull request merged**